### DISCUSSION TITLE
Unable to build micronaut-core locally

### DISCUSSION THREAD

**Discussion by pragyamutluru** [micronaut-projects/micronaut-core#10988] [Q&A] [answered]
Hello everyone!

I am new to the project and am planning to make my first contribution soon.
I took a pull of the 4.6x branch and tried to build it locally.

However, I am facing an error :

"* What went wrong:
An exception occurred applying plugin request [id: 'io.micronaut.build.internal.docs']
> Failed to apply plugin class 'io.micronaut.build.docs.JavadocAggregatorPlugin'.
   > A problem occurred configuring project ':test-suite-geb'.
      > Could not resolve all artifacts for configuration ':test-suite-geb:classpath'.
         > Could not find org.ysb33r.gradle:grolifant:0.5.1.
           Searched in the following locations:
             - https://plugins.gradle.org/m2/org/ysb33r/gradle/grolifant/0.5.1/grolifant-0.5.1.pom
             - https://repo.maven.apache.org/maven2/org/ysb33r/gradle/grolifant/0.5.1/grolifant-0.5.1.pom
           Required by:
               project :test-suite-geb > gradle.plugin.com.energizedwork.webdriver-binaries:webdriver-binaries-gradle-plugin:1.4"

Before building the project, should I manually make some changes in build.gradle or settings.xml?
I could not find any steps relevant to it in 
https://github.com/micronaut-projects/micronaut-core/blob/864d3c08940ab4b5a6f0514ca1c7ec895976ec45/CONTRIBUTING.md

Any help/suggestion would be appreciated!

**Comment by yawkat** [comment]
I think this may be due to https://blog.gradle.org/portal-jcenter-impact , @melix

**Reply by melix** [reply, marked as answer]
PR https://github.com/micronaut-projects/micronaut-core/pull/10990
